react/Dialog/index.jsx

```diff
@@ -21,12 +23,47 @@ const Dialog = props => {
     (props.open || props.opened) && shouldBlockScroll
       ? RemoveScroll
       : React.Fragment
+  const cozyTheme = useCozyTheme()
+  const theme = useTheme()
+  const cozBar = document.querySelector('.coz-bar-wrapper')
+  const sidebar = document.querySelector('[class*="sidebar"]')
 
-  const theme = useCozyTheme()
+  useSetFlagshipUI(
```

Comment:
nit: add a test checking useSetFlagshipUI is called with correct arguments, when this component is rendered

Reply:
added more tests https://github.com/cozy/cozy-ui/pull/2072/files#diff-4cbb730f55f4a37e3fa25231e3f79b6f1d5901fa59e89ec6455568fc400be2fdR1